### ISSUE TITLE
Log raid nukes with ceilling and floor

### ISSUE THREAD

**Issue by almostSouji** [Naval-Base/yuudachi#1090]
- cases currently only support `join_cutoff` and `account_cutoff`, however, this is no longer how the command works. we now define floors and ceilings for creation and join dates, each being optional
- needs a solid migration (cutoffs are after* parameters)
- needs adaptation of logging (including i18n)
- needs adaption of creating the case

> **Note**
> https://github.com/Naval-Base/yuudachi/pull/1089 moves core nuke logic and execution into a centralized function and should be merged before attempting this migration, as it simplifies the process

**Comment by JPBM135** [comment]
Why we register cuttoff in the case? Now with the archive feature (markdown soonTM) this don't need to be the case anymore.

**Comment by almostSouji** [comment]
good point, removing that metric is a valid approach, when adding the archive

**Comment by iCrawl** [comment]
`join_cutoff` and `account_cutoff` is not removed from cases and case logs.

**Comment by iCrawl** [comment]
Should have a separate ticket for any other additions.
